test/plugin/scenarios/sofarpc-scenario/config/expectedData.yaml

```diff
@@ -34,6 +51,37 @@ segmentItems:
         refType: CrossProcess, parentSpanId: 1, parentTraceSegmentId: not null, parentServiceInstance: not
           null, parentService: sofarpc-scenario, traceId: not null}
       skipAnalysis: 'false'
+  - segmentId: not null
```

Comment:
> Is expectedData.yaml file strongly depends on the stop order of each span? I am using async invoke ,so the stop order is not fixed. It may affect CI success rate.

Considering this is a new segment, which part of the span order are you referring to?

Reply:
![image](https://github.com/apache/skywalking-java/assets/13081689/edf53e2e-dcf4-4f71-841c-90b403c02cfc)

I mean main thread segment and callback thread segment.  Which segment will stop first is not fixed.

I try to sleep in main thread to make main thread segment stop after callback thread segment. The test runs successfully. So, segment order in expectedData.yaml is ignored?

Reply:
Segment order is not a part of validation in the file. You don't need to worry about it.